docs/tutorial/node/src/main_game_logic.js

```diff
@@ -0,0 +1,66 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+
+// main.js
+let slint = require("slint-ui");
+let ui = require("./memory.slint");
+let window = new ui.MainWindow();
```

Comment:
In the text we refer to the window as `main_window`. I think this should be renamed here, also to disambiguate with the global "window" folks may know from the browser.